src/platforms/shared.rs

```diff
@@ -82,3 +81,21 @@ pub fn shared_build_opts_ccache(use_ccache: bool) -> Vec<String> {
         vec![]
     }
 }
+
+/// Ignore duplicate libraries warnings for MacOS with XCode>=15.
+pub fn macos_build_opts_ignore_dupicate_libs_warnings() -> Vec<String> {
+    let xcode_version =
+        crate::utils::get_xcode_version().unwrap_or(crate::utils::XCODE_MIN_VERSION);
+    if xcode_version >= 15 {
+        println!(
```

Comment:
`eprintln!` please!

Reply:
This is not an error, this is an info message. But I believe we don't print anything like that in our tool, so, I just removed this.

Reply:
@antonbaliasnikov yeah, that may sound counter-intuitive.
In our binaries we usually print all diagnostic messages and logs to stderr, and reserve stdout for output with a potentially pre-defined format, with the most common case as solc/zksolc standard JSON I/O.
So the parent process can always parse stdout, and simply bubble-up stderr if exit code is non-zero.